packages/contracts-bedrock/contracts/L2/L2ERC721Bridge.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Shouldn't this just appear as a file rename from the periphery package?

Reply:
git is not catching this for some reason, there must be a large enough diff in the file - I used `diff` on each to double check that the only diffs are in the imports